tronweb/coffee/models.coffee

```diff
@@ -54,6 +54,16 @@ class window.RefreshModel extends Backbone.Model
             console.log("scheduled with " + @interval)
             @timeout = setTimeout(@doRefresh, @interval)
 
+class window.StatusModel extends Backbone.Model
+
+    urlRoot: ->
+        "/status/"
+
+    parse: (resp) =>
+        booted = moment.unix(resp['boot_time']).format('YY-MM-DD HH:mm ZZ')
```

Comment:
I think YYYY would make the date easier to read. Otherwise, the formatting looks good to me.

Reply:
https://www.reddit.com/r/ISO8601/

Reply:
Sure, I was thinking about trying to shorten this since I'd assume there is no tron server booted in 19XX, but keeping consistent formatting is probably more valuable.